elasticsearch/src/main/java/com/gentics/mesh/search/index/node/NodeSearchHandler.java

```diff
@@ -83,6 +84,7 @@ public NodeSearchHandler(SearchProvider searchProvider, Database db, NodeIndexHa
 		if (log.isDebugEnabled()) {
 			log.debug("Invoking search with query {" + query + "} for {Containers}");
 		}
+		// TODO !! pass along type parameter to reduce / correct selected indices
```

Comment:
Override getIndexHandler in NodeSearcHandler. Implement `getIndicesForSearch` with additional type as parameter